Ginger/GingerCore/GeneralLib/GenericWindow.xaml.cs

```diff
@@ -219,9 +219,9 @@ public partial class GenericWindow : Window
 
             //close buttons handling
             mCloseEventHandler = closeEventHandler;
-            if (!showClosebtn)
+            if (!dontShowCloseBtn)
```

Comment:
Why are we changing it to be visible when its visible by default, also please check other places wherever this window is being used

Reply:
I checked the xaml and by default it is Collapsed. I checked other places and every 
"true" or no attribute is passed for "showCloseBtn"